wheatley/tower.py

```diff
@@ -142,9 +142,24 @@ def _create_client(self):
         self._socket_io_client.on("s_size_change", self._on_size_change)
         self._socket_io_client.on("s_assign_user", self._on_assign_user)
         self._socket_io_client.on("s_call", self._on_call)
+        self._socket_io_client.on("s_user_left", self._on_user_leave)
 
         self._request_global_state()
 
+    def _on_user_leave(self, data):
+        user_that_left = data["user_name"]
+
+        bells_unassigned = []
+
+        # Unassign all instances of that user
+        for i in self._assigned_users.keys():
+            if self._assigned_users[i] == user_that_left:
+                self._assigned_users[i] = None
+
+                bells_unassigned.append(i.number)
```

Comment:
```suggestion
        for bell, user in self._assigned_users.items():
            if user == user_that_left:
                self._assigned_users[bell] = None

                bells_unassigned.append(str(bell))
```

`str(bell)` gives `0`, `E`, `T`
And this makes it a bit clearer what you're iterating over

Reply:
Sounds good - I'll do it when I get back to my laptop. Wouldn't it also make sense to override `__repr__` for the `Bell` class?  This way it doesn't print something annoying like `<class wheatley.bell.Bell at xxxxxx>` when put inside a list.